manim/mobject/mobject.py

```diff
@@ -2135,6 +2135,11 @@ def __len__(self):
     def get_group_class(self):
         return Group
 
+    @staticmethod
+    def get_mobject_type_class():
+        """Return the base class of this mobject type."""
+        return Mobject
+
```

Comment:
If you want a more generic/automatic way of doing this, it may be helpful to look at #1353 for inspiration, where you can do

```py
class MyMobjectType(Mobject, mobject_type=True):
    ...
```

Then you would be able to just access `MyMobjectType.mobject_type` which would cascade down to subclasses.

You could also just do
```py
class MyMobjectType(Mobject):
    ...

MyMobjectType.mobject_type = MyMobjectType
```

But that seems worse to me (albeit perhaps simpler mechanically).